lib/Backends/OpenCL/kernels.cl

```diff
@@ -22,6 +22,13 @@ typedef struct {
   cl_uint64_t w; // Width
 } ShapeNCHW;
 
+typedef struct {
```

Comment:
Please don't use size_t in OpenCL kernels. Its size is target dependent. Use cl_uint64_t instead, like other structs in this file.